odoo/service/server.py

```diff
@@ -443,9 +443,9 @@ def worker_spawn(self, klass, workers_registry):
 
     def long_polling_spawn(self):
         nargs = stripped_sys_argv()
-        cmd = os.path.join(os.path.dirname(nargs[0]), "odoo.py")
-        nargs = [cmd, 'gevent'] + nargs[1:]
-        popen = subprocess.Popen([sys.executable] + nargs)
+        parent = os.path.dirname(config['root_path'])
+        cmd = [sys.executable, '-m', 'odoo', 'gevent'] + nargs[1:]
```

Comment:
@KangOl @sle-odoo in my original PR, it was this commit: https://github.com/odoo/odoo/pull/13237/commits/55be6763611c7ea51a7350be9b290cf4b57d67f9